src/analysis/scripts/export_connectivity.sh

```diff
@@ -75,6 +75,35 @@ function ec_export_table_csv() {
          -c "\COPY ${EXPORT_TABLENAME} TO '${FILENAME}' WITH (FORMAT CSV, HEADER)"
 }
 
+function ec_export_table_geojson() {
+  OUTPUT_DIR="${1}"
+  EXPORT_TABLENAME="${2}"
+
+  FILENAME="${OUTPUT_DIR}/${EXPORT_TABLENAME}.geojson"
+  ogr2ogr -f GeoJSON "${FILENAME}" \
+          -t_srs EPSG:4326 \
```

Comment:
I ran into an error where for some of the destinations tables ogr2ogr threw an error about not finding appropriate geometries. I can remove it and test again, maybe the error was fixed via further munging of these commands.